extensions/arc/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "arc",
   "displayName": "%arc.displayName%",
   "description": "%arc.description%",
-  "version": "0.12.0",
+  "version": "1.0.0",
```

Comment:
You also should be bumping up the azdata engine version, right? Since you rely on latest changes in resource-deployment.

Reply:
Good point, just pushed changes to make azdata version requirement 1.35.